.env.development.EXAMPLE

```diff
@@ -1,3 +1 @@
-GATSBY_ALGOLIA_APP_ID=""
-GATSBY_ALGOLIA_SEARCH_KEY=""
-GATSBY_ALGOLIA_INDEX="businesses"
\ No newline at end of file
+SEARCH_API_ENDPOINT=''
```

Comment:
This may be me doing something incorrectly but I _think_ that the env variables used on the frontend need to be prefixed with `GATSBY_` (similar to how we had the Algolia env vars set up).

Reply:
Oh duh! I totally forgot that! I'll do that in a bit!